### PR TITLE
Linux 下在创建快捷方式时使用 xdg-user-dir 寻找桌面地址

### PR DESCRIPTION
**Pull request by CareF**
目前的代码只能处理中文和英文系统的默认地址, 不能处理用户自定义的地址和其他语言地址.

### CODE REVIEW

File: feeluown/cli/install.py [hunk 1]
```diff
@@ -80,16 +80,16 @@ def gen_for_win_linux():
...
+        desktop_path_str = os.popen('xdg-user-dir DESSKTOP').read().split()[0]
+        assert desktop_path_str
+        desktop_path = pathlib.Path(desktop_path_str)
+        assert desktop_path.exists()
+        desktop_file = desktop_path / DESKTOP_FILE
         shutil.copy(f_path, desktop_file)
-    os.system('chmod +x {}'.format(desktop_file))
+        os.system('chmod +x {}'.format(desktop_file))
+    except (FileNotFoundError, IndexError, AssertionError):
+        pass
```
Comment (cosven) [inline feeluown/cli/install.py:92]: Raise a error when failed.

File: feeluown/cli/install.py [hunk 2]
```diff
@@ -80,16 +80,16 @@ def gen_for_win_linux():
...
-    en_desktop_path = pathlib.Path.home() / 'Desktop'
-    cn_desktop_path = pathlib.Path.home() / '桌面'
-    desktop_file = None
-    if en_desktop_path.exists():
-        desktop_file = en_desktop_path / DESKTOP_FILE
-    if cn_desktop_path.exists():
-        desktop_file = cn_desktop_path / DESKTOP_FILE
-    if desktop_file:
+    try:
+        desktop_path_str = os.popen('xdg-user-dir DESSKTOP').read().split()[0]
```
Comment (cosven) [inline feeluown/cli/install.py:84]: `DESSKTOP` typo?

File: feeluown/cli/install.py [hunk 3]
```diff
@@ -80,16 +80,16 @@ def gen_for_win_linux():
...
+        assert desktop_path_str
+        desktop_path = pathlib.Path(desktop_path_str)
+        assert desktop_path.exists()
+        desktop_file = desktop_path / DESKTOP_FILE
         shutil.copy(f_path, desktop_file)
-    os.system('chmod +x {}'.format(desktop_file))
+        os.system('chmod +x {}'.format(desktop_file))
```
Comment (cosven) [inline feeluown/cli/install.py:90]: Can we put these two lines outside the try...except... block?
```
shutil.copy(f_path, desktop_file)
os.system('chmod +x {}'.format(desktop_file))
```

File: feeluown/cli/install.py (hunk 1 above)
Comment (CareF) [reply feeluown/cli/install.py:92]: This is to keep consistent with previous behavior that when the desktop dir is not found, do nothing.

File: feeluown/cli/install.py (hunk 3 above)
Comment (CareF) [reply feeluown/cli/install.py:90]: Sure but similiar to https://github.com/feeluown/FeelUOwn/pull/473#discussion_r611125200

File: feeluown/cli/install.py (hunk 2 above)
Comment (CareF) [reply feeluown/cli/install.py]: Thanks. Fixed.

**Review by cosven — APPROVED** [review]
LGTM

**Pull request merged**